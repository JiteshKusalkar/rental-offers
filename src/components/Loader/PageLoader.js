import PropTypes from 'prop-types';
import { Overlay, Wrapper } from './styles';

function PageLoader({ children }) {
  return (
    <Wrapper>
      <Overlay />
      {children}
    </Wrapper>
  );
}

PageLoader.propTypes = {
  children: PropTypes.node,
};

export default PageLoader;
