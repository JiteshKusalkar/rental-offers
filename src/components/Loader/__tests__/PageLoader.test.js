import { render } from '@testing-library/react';
import PageLoader from '../PageLoader';

describe('PageLoader', () => {
  it('should render the component', () => {
    const { container } = render(<PageLoader>Card content</PageLoader>);

    expect(container).toMatchSnapshot();
  });
});
