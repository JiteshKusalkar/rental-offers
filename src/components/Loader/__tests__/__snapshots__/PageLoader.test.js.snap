// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoader should render the component 1`] = `
<div>
  <div
    class="sc-bdvvtL ibNRhI"
  >
    <div
      class="sc-gsDKAQ fdzcKP"
    />
    Card content
  </div>
</div>
`;
